Source/WebCore/css/CSSPageRule.cpp

```diff
@@ -57,10 +57,10 @@ String CSSPageRule::selectorText() const
 {
     if (auto* selector = m_pageRule->selector()) {
         String pageSpecification = selector->selectorText();
-        if (!pageSpecification.isEmpty() && pageSpecification != starAtom())
-            return makeString("@page "_s, pageSpecification);
+        if (!pageSpecification.isEmpty())
+            return pageSpecification;
```

Comment:
Given the `return ""_s` below doesn't this simplify to `return selector->selectorText()`?

Reply:
Update - noted, we need to return initially `emptyString()` so I kept it `""_s`.